### ISSUE TITLE
Checking for updates does not work

### ISSUE THREAD

**Issue by koxu1996** [JGillam/burp-co2#29]
I tried version 1.1.11 from BAppStore and jar builded from latest code, but checking for updates does not work for both:
![peek 2017-09-24 09-37](https://user-images.githubusercontent.com/6506780/30780492-560f5b1c-a10c-11e7-9eb5-c2361fdf5522.gif)

**Comment by koxu1996** [comment]
I figured out the problem: http://www.burpco2.com is redirecting to http://burpco2.com, so response code is _301 Moved Permamently_, not _200 OK_ as expected:
```
byte[] response = callbacks.makeHttpRequest("www.burpco2.com", 80, false, request);
IResponseInfo responseInfo = callbacks.getHelpers().analyzeResponse(response);
if (responseInfo.getStatusCode() == 200) {
```

**Comment by JGillam** [comment]
Fixed in version 1.1.12
